modules/acm/src/main/java/edu/gemini/epics/acm/CaApplySenderImpl.java

```diff
@@ -29,7 +31,7 @@
     private final CaApplyRecord apply;
     private final CaCarRecord<C> car;
 
-    private final Boolean trace = false;
+    private final Boolean trace = true;
```

Comment:
`Boolean.getBoolean("epics.apply.trace")` ?

Reply:
Yes, you're right